src/Mono.Android/Test/Mono.Android-Tests.projitems

```diff
@@ -4,7 +4,9 @@
     <UnitTestApk Include="$(OutputPath)Mono.Android_Tests-Signed.apk">
       <Package>Mono.Android_Tests</Package>
       <InstrumentationType>xamarin.android.runtimetests.TestInstrumentation</InstrumentationType>
-      <ResultsPath>$(MSBuildThisFileDirectory)..\..\..\TestResult-Mono.Android_Tests.xml</ResultsPath>
+      <ResultsPath Condition=" '$(Configuration)' != 'Release' " >$(MSBuildThisFileDirectory)..\..\..\TestResult-Mono.Android_Tests.xml</ResultsPath>
```

Comment:
I considered similar name, but because AotAssemblies is usually empty I opted for the conditions. If you prefer the names ending with `+Aot=.xml` I can change it. Or we can have 2 cases with and without Aot in the name. Like `TestResult-Mono.Android_Tests-$(Configuration).xml` and `TestResult-Mono.Android_Tests-$(Configuration)-Aot.xml`. What do you think?